src/server/views/dashboard/macros.njk

```diff
@@ -11,8 +11,8 @@
   </li>
 {% endmacro %}
 
-{% macro successBanner(bannerHeading = "Success", bannerTitle = "Saved successfully", colourClass = ".govuk-tag--green") %}
-  <div class="govuk-notification-banner {{ colourClass }}" role="alert" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
```

Comment:
cc @alisalaman-cyb - I think you originally implemented. 

fwiw - it was also passing in `.govuk-tag` into "class". but it should've been `govuk-tag`. (i.e. without the leading `.`)